workflow-templates/localgov-drupal-ci.yml

```diff
@@ -26,21 +26,25 @@ jobs:
       - name: Create LocalGov Drupal project
         run: composer create-project --stability dev localgovdrupal/localgov-project html
 
-      - name: Setup package source for the test target
-        run: |
-          composer --working-dir=html config repositories.1 vcs git@github.com:${{ github.repository }}.git
-          composer global config github-oauth.github.com ${{ github.token }}
-
-      - name: Extract Git branch name outside of a pull request
+      - name: Save git branch and git repo names to env if not a pull request
         if: github.event_name != 'pull_request'
-        run: echo "GIT_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+        run: |
+          echo "GIT_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+          echo "GIT_REPO=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
 
-      - name: Extract Git branch name from a pull request
+      - name: Save git branch and git repo names to env if is a pull request
```

Comment:
Err... there's a small typo :( @tebb

Reply:
Thanks for reviewing Adnan.

Not quite sure what you want.  I was trying to keep it short and did mean 'is'???

I'll change to :  Save git branch and git repo names to environment if this is a pull request

... and change the similar, adjacent comment to match.